lua/nvim-treesitter/query.lua

```diff
@@ -89,6 +89,11 @@ function M.get_query_files(lang, query_name)
   local lang_files = filtered_runtime_queries(lang, query_name)
   vim.list_extend(query_files, lang_files)
 
+  for _, base_lang in ipairs(M.base_language_map[lang] or {}) do
```

Comment:
I think this should move above the `lang_files` so base files are loaded before the standard lang files.